tools/packaging/tropohouse.js

```diff
@@ -304,7 +304,7 @@ _.extend(exports.Tropohouse.prototype, {
     // it relies on extractTarGz being fast and not reporting any progress.
```

Comment:
It's just off the top of the diff here, but I think this `XXX/TODO` would be a huge benefit as it's getting hard to debug when something is stuck `downloading` or something is stuck `extracting`.

I'm getting the feeling that people are getting stuck _after_ the actual download, especially on Windows.